src/Modules/Backend/Backend/Actions/ModuleSettings.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace ForkCMS\Modules\Backend\Backend\Actions;
+
+use ForkCMS\Core\Domain\Header\FlashMessage\FlashMessage;
+use ForkCMS\Core\Domain\Kernel\Command\ClearContainerCache;
+use ForkCMS\Modules\Backend\Domain\Action\AbstractFormActionController;
+use ForkCMS\Modules\Backend\Domain\ModuleSettings\ModuleSettingsType;
+use ForkCMS\Modules\Backend\Domain\User\User;
+use ForkCMS\Modules\Backend\Domain\User\UserRepository;
+use ForkCMS\Modules\Extensions\Domain\Module\Command\ChangeModuleSettings;
+use ForkCMS\Modules\Extensions\Domain\Module\Module;
+use ForkCMS\Modules\Extensions\Domain\Module\ModuleName;
+use RuntimeException;
+use Symfony\Component\Form\FormInterface;
+use Symfony\Component\HttpFoundation\RedirectResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+
+final class ModuleSettings extends AbstractFormActionController
+{
+    protected function getFormResponse(Request $request): ?Response
+    {
+        $moduleRepository = $this->getRepository(Module::class);
+        $moduleName = $this->getModuleName();
+
+        return $this->handleSettingsForm(
+            request: $request,
+            formType: ModuleSettingsType::class,
+            formData: new ChangeModuleSettings(
+                $moduleRepository->find(ModuleName::core()) ?? throw new RuntimeException('Core module not found'),
+                $moduleRepository->find($moduleName) ?? throw new RuntimeException($moduleName . ' module not found'),
+                []
+            ),
+            validCallback: function (FormInterface $form): Response {
```

Comment:
This looks like code that belongs in the command handler. clearing the cache by throwing the event

Reply:
fixed f0c5e68ddfcdf86877dc7596bdf9c6258f6ddaa2